SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -35,6 +38,11 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 public:
 	virtual void FinishDestroy() override;
 	void DestroyConnection();
+    
+    /// Caller can register an callback by using this function.
+    /// It will be trigered when SpatialWorkerConnection receive Login Token.
+    /// @param cb - callback function.
+    void RegisterOnLoginTokensCb(const LoginTokenCb& cb) {LoginTokenCb_ = cb;};
```

Comment:
Probably want to align "cb" with the variable coding standard, having a capital at that start. Perhaps just Callback would do.